Specs/Scene/BatchTableHierarchySpec.js

```diff
@@ -498,4 +602,30 @@ describe("Scene/BatchTableHierarchy", function () {
       });
     }).toThrowDeveloperError();
   });
+
+  it("Computes memorySizeInBytes from typed arrays", function () {
```

Comment:
This is the test that fails when run with other tests.

Consider cloning the extension JSON and binary payload to see if they're getting overwritten.

Reply:
As I figured, `getBinaryProperties()` modifies the JSON in place. I changed the unit tests to deep copy the JSON, that fixes the test.